### PR TITLE
Feature/oauth servers render list

### PR DESCRIPTION
**Pull request by ivanzusko**

### CODE REVIEW

File: src/modules/pages/NewOAuthServerPage/OAuthServerForm.js [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+const selector = formValueSelector('oAuthServerForm');
+
+const form = reduxForm({
+    form: 'oAuthServerForm',
+    enableReinitialize: true, // this is needed!!
+})(OAuthServerForm);
+
+export default connect(
+    // state => {
```
Comment (rafaeljesus) [inline src/modules/pages/NewOAuthServerPage/OAuthServerForm.js]: 🤔

Comment (ivanzusko) [reply src/modules/pages/NewOAuthServerPage/OAuthServerForm.js]: yeap, sorry 😞  removed. Redundant as a connect at this moment.

Comment (rafaeljesus) [reply src/modules/pages/NewOAuthServerPage/OAuthServerForm.js]: I didn't follow sorry, what you meant?

**Comment by rafaeljesus** [comment]
👍

**Pull request merged**